validate.go

```diff
@@ -170,6 +170,17 @@ func checkProcess(process rspec.Process, rootfs string) {
 		}
 	}
 
+	if len(process.Args) < 1 {
+		logrus.Fatalf("args should not be empty")
+	}
+	command := process.Args[0]
+	if filepath.IsAbs(command) {
+		cmdPath := path.Join(rootfs, command)
+		if _, err := os.Stat(cmdPath); err != nil {
+			logrus.Fatalf("Cannot find the command path %q", cmdPath)
```

Comment:
This check is too strict.  Perhaps the user intends to copy the executable into the right location before running `start` (see earlier discussion [here][1]).  Or maybe the user will bind-mount the command in from somewhere else.  Anyhow, I think it's to complicated to try and make this a validation-time check, and we should just leave it to the runtime/kernel to error out if the command isn't there at execve-time (or whatever syscall the runtime is triggering with ‘start’).

[1]: https://github.com/opencontainers/runc/pull/827#discussion_r63464201